packages/cosmos_ui_components/lib/utils/date_formatter.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:intl/intl.dart';
+
+String formatDate(DateTime date, DateFormatEnum dateFormat) => DateFormat(getDateFormat[dateFormat]).format(date);
+
+abstract class DateFormats {
+  static const String UIShortDate = 'MMM d';
```

Comment:
variables should start with lowercase letters:
```suggestion
  static const String uiShortDate = 'MMM d';
```